shell/platform/android/android_surface_software.cc

```diff
@@ -16,6 +16,23 @@
 
 namespace shell {
 
+namespace {
+
+bool getSkColorType(int32_t buffer_format, SkColorType* color_type) {
```

Comment:
Nit: I forget how we name out static methods. But isn't it something like `GetSkColorType`?